src/app/blueprint.css

```diff
@@ -1,50 +1,57 @@
-/* Styles for root page */
-.page {
-  font-family: system-ui, "Segoe UI", Roboto, Helvetica, Arial, sans-serif,
-    "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+body {
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
+    Ubuntu, Cantarell, 'Helvetica Neue', sans-serif;
 }
 
-/* Styles for feature component */
-
 .feature {
-  background-color: #ffffff;
+  display: flex;
+	justify-content: center;
+  max-width: calc(100% - 2rem);
+  margin: 3rem auto;
+  padding: 3rem;
+	background-color: white;
   border: 1px solid #dbdde2;
```

Comment:
Could we put all these colors in a CSS variables? I think the coverage is fair, it's already the default choice for developers and would eliminate repetition.